src/components/Box/__tests__/box-tests.jest.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { fireEvent, render, cleanup } from "@testing-library/react";
+import { act } from "@testing-library/react-hooks";
+import Box from "../Box";
+
+
```

Comment:
there is no need in behavior tests for this component

Reply:
+1

Reply:
removed not needed.

Reply:
but  there are still tests here? the snapshot check existence of class name